src/app/subscriptions.ts

```diff
@@ -17,69 +17,35 @@ import { AppStore } from "./state/store";
 import { rewardSlice } from "./state/rewardSlice";
 
 export type RDT = ReturnType<typeof RadixDappToolkit>;
-
-let rdtInstance: null | RDT = null;
-export function getRdt() {
-  return rdtInstance;
-}
-
-export function getRdtOrThrow() {
-  const rdt = getRdt();
-  if (!rdt) {
-    throw new Error("RDT initialization failed");
-  }
-  return rdt;
-}
-function setRdt(rdt: RDT) {
-  rdtInstance = rdt;
-}
+export let rdt: RDT;
 
 let subs: Subscription[] = [];
+let shortInterval: NodeJS.Timer;
 
 export function initializeSubscriptions(store: AppStore) {
-  let networkId;
-  switch (process.env.NEXT_PUBLIC_NETWORK!) {
-    case "mainnet":
-      networkId = RadixNetwork.Mainnet;
-      break;
-    case "stokenet":
-      networkId = RadixNetwork.Stokenet;
-      break;
-    default:
-      networkId = RadixNetwork.Stokenet;
-  }
-  rdtInstance = RadixDappToolkit({
+  rdt = RadixDappToolkit({
     dAppDefinitionAddress: process.env.NEXT_PUBLIC_DAPP_DEFINITION_ADDRESS!,
-    networkId,
+    networkId:
+      process.env.NEXT_PUBLIC_NETWORK == "mainnet"
```

Comment:
```suggestion
      process.env.NEXT_PUBLIC_NETWORK! == "mainnet"
```
This non-null TS assertion just ensures we have more readable errors if the env var is not set up.

Reply:
Yep, agree. Must have missed that one.